src/cal/test_atmos_corr_raw_to_rad.py

```diff
@@ -2,21 +2,36 @@
 Programmer: Brian Szutu

```

Comment:
Well done for the banner! It really helps. Do you mind changing it in the following way:

```
"""
Author: Your Name
email: your email (optional)
License: MIT
Copyright: 2018-2019
"""
```